backend/__init__.py

```diff
@@ -0,0 +1,15 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+import os
+
+db_role = 'postgres'
+db_password = 'edison'
+db_url = '0.0.0.0'
```

Comment:
should this be 0.0.0.0 or 127.0.0.1?

Reply:
Since our app is running on 0.0.0.0 we think the db should run on the same IP.
We tried change both IPs to 127.0.0.1 but somehow we couldn't reach the server on the host machine. Furthermore, only when app IP is 0.0.0.0 we could reach it through localhost on the host machine.

Reply:
> your should really figure out a way to ensure your DB is not accessible to anything other
> then your App, the best technique is to actually not make it accessible over IP at all, and
> use a local UNIX socket instead.

Understood. I could change db IP to 127.0.0.1 as proposed above and the db won't be accissible outside the VM.

Should I also change flask servers IPs to something different than 0.0.0.0 ?

Reply:
Well, the flask servers run your user-facing applications, so it makes sense to run them in a way that makes them accessible to users.

Reply:
ack